mmv1/templates/terraform/pre_create/dialogflow_set_location.go.erb

```diff
@@ -0,0 +1,13 @@
+
+// extract location from the parent
+stringParts := strings.Split(d.Get("parent").(string), "/")
```

Comment:
Do we know all parents have this format? We could use a regex to capture `projects/.*/locations/.*/(capture group)/.*`

Reply:
sure let me try this

Reply:
Ok I replaced the function to use a regex extraction instead